commands/tenants

```diff
@@ -27,27 +31,37 @@ create_subtenant() {
   local domain="$1"
   local username="$2"
   local password="$3"
+  local email="$4"
   shift # Remove domain from $@
   shift # Remove username from $@
   shift # Remove password from $@
+  shift # Remove email from $@
+
+  if [[ -z "$company" ]]; then
+    company="SAG"
+  fi
+  if [[ -z "$contactName" ]]; then
+    contactName="IoT Extensions"
+  fi
 
   if [[ "$ignoreAlreadyDone" == true ]]; then
-    response=$(c8y tenants create --domain "$domain" --adminName "$username" --adminPass "$password" --company 'SAG' --contactName 'IoT Extensions' --data 'adminEmail=iot-analytics-mobile-accelerator@softwareag.com' --timeout 300000 --silentStatusCodes=409 "$@")
+    response=$(c8y tenants create --domain "$domain" --adminName "$username" --adminPass "$password" --company "$company" --contactName "$contactName" --data "adminEmail=$email" --timeout 300000 --silentStatusCodes=409 "$@")
```

Comment:
- to avoid code duplication with the lines in the else block: is it possible to "dynamically" add the `--silentStatusCodes=409` as string to the command?